### ISSUE TITLE
A small bug in reparameterize() 

### ISSUE THREAD

**Issue by elokuuM** [AILab-CVC/UniRepLKNet#8]
Problem with parameter in unireplknet.py line 338.
             conv = nn.Conv2d(self.dwconv.in_channels, self.dwconv.out_channels, self.dwconv.kernel_size,
                                 self.dwconv.padding, self.dwconv.groups, bias=True)

->             conv = nn.Conv2d(self.dwconv.in_channels, self.dwconv.out_channels, self.dwconv.kernel_size,
                                 padding=self.dwconv.padding, groups=self.dwconv.groups, bias=True)

**Comment by elokuuM** [comment]
And, in unireplknet.py line 149.
identity_kernel = torch.ones((1, 1, 1, 1))

-> identity_kernel = torch.ones((1, 1, 1, 1)).to(kernel.device)

**Comment by DingXiaoH** [comment]
Thank you for the valuable feedback! Fixed that!
